kiwi/storage/subformat/vagrant_libvirt.py

```diff
@@ -15,63 +15,26 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-import json
 import os
-import random
+from typing import Dict  # pylint: disable=unused-import
```

Comment:
why, unused imports should be deleted

Reply:
That was necessary for the typing comment: `# type: () -> Dict[str, str]`.

This is an unfortunate consequence of using type hints in a python 2 compatible code base: you have to import all the types so that a type checker can evaluate these, but since you only use them in comment blocks, pylint & co. complain about unused imports since they don't evaluate type hints.